oper8/dag/node.py

```diff
@@ -129,23 +133,33 @@ def api_group(self) -> str:
     @property
     def api_version(self) -> str:
         """The full kubernetes apiVersion"""
-        return self.get_data().get("apiVersion")
+        return self.manifest.get("apiVersion")
 
     @property
     def kind(self) -> str:
         """The resource kind"""
-        return self.get_data().get("kind")
+        return self.manifest.get("kind")
 
     @property
     def metadata(self) -> dict:
         """The full resource metadata dict"""
-        return self.get_data().get("metadata", {})
+        return self.manifest.get("metadata", {})
 
     @property
     def name(self) -> str:
         """The resource metadata.name"""
         return self.metadata.get("name")
 
+    @property
+    def manifest(self) -> str:
+        """The resource manifest"""
+        return self.get_data()
+
+    @property
+    def verify_function(self) -> str:
```

Comment:
Crap, one more NIT: The return on this should be `Callable`

Reply:
Good catch! I will also fix the manifest section as well